### ISSUE TITLE
Real avatars

### ISSUE THREAD

**Issue by magicznyleszek** [kobotoolbox/kpi#2244]
## Description

It would be nice to have real avatars, i.e. uploadable by users. We can display them in header, in #2183, new public collections and possibly more.

## Actual behavior 

We have a HTML avatars build with coloured circle and user initials.

## Additional details

We used gravatar, but:

> Re:avatars, some history on why we don’t have them now: we used to have gravatar integration (via email), and there were some privacy concerns, and some users found it odd that we used their avatar without them uploading any files or explicitly giving use permission to use their avatar.

**Comment by magicznyleszek** [comment]
@jnm @noliveleger any blockers here?

**Comment by magicznyleszek** [comment]
Blocked by #2626
